### PR TITLE
Move date-fns to development dependencies and update dependencies

### PR DESCRIPTION
**Pull request by bttger**
There were some vulnerabilities and the versions in the package.json file were fixed. I recommend using the caret versioning to be able to easily update to new versions or auto-fix npm audits without breaking changes.

I have checked the changelogs of the updated packages and ran the tests after updating.

### CODE REVIEW

**Comment by TimDaub** [comment]
> I recommend using the caret versioning to be able to easily update to new versions or auto-fix npm audits without breaking changes.

No we're not doing that. It allows then for NPM supply chain attacks so all packages must be pinned.

**Comment by TimDaub** [comment]
Your fix is conceptually valuable but your're using the caret so I'll close this and move date-fns into devdepencencies. thanks anyways.

**Comment by TimDaub** [comment]
merging it for credit

File: package.json [hunk 1]
```diff
@@ -26,11 +26,9 @@
   },
   "homepage": "https://github.com/TimDaub/better-sqlite3-session-store#readme",
   "devDependencies": {
-    "ava": "3.13.0",
-    "better-sqlite3": "7.1.1",
-    "express-session": "1.17.1"
-  },
-  "dependencies": {
-    "date-fns": "2.16.1"
+    "ava": "^4.3.3",
```
Comment (TimDaub) [inline package.json:29]: as a feedback, you should never "just" upgrade a major version number as it contains breaking changes.

Comment (TimDaub) [reply package.json:29]: here's the fix for it: 86a4d14

**Comment by bttger** [comment]
Thank you a lot for bringing the supply chain issue to my attention and for merging it.

**Pull request merged**